### DISCUSSION TITLE
What's happening to React Native

### DISCUSSION THREAD

**Discussion by moishinetzer** [remix-run/react-router#11655] [Q&A] [answered]
In the React Native scene, there aren't any real solutions around complex data loading. Nearly all projects use TanStack Query because there are no deeper integrated options out there.

Docs:
> The data APIs from v6.4 are currently not supported in React Native, but should be eventually.

With the work on V7 currently happening is this still in the books? It would be revolutionary for React Native.

**Comment by sergiodxa** [comment, marked as answer]
The team mentioned on Discord that react-router-native is going to be removed and not supported anymore. The community never used RR on native and went with other routes that leverage native routes, so the team will focus on the web only.

With RSC coming to RN that may be a solution for data loading in RN apps.

**Comment by moishinetzer** [comment]
That's a shame but it's nice to see a clear focus on what the goals are. It would be nice to see the docs fixed to reflect this. Happy to put in a PR but I feel like this is something the team will want to address with their own language?

**Reply by kiliman** [reply]
Also, it seems like Expo Router will become the de facto routing solution for React Native.
